app/controllers/lit/sources_controller.rb

```diff
@@ -20,7 +20,12 @@ def edit
 
     def synchronize
       @source = Source.find(params[:id])
-      @source.synchronize
+      @source.update_column(:sync_complete, false)
+      if defined?(ActiveJob)
+        SynchronizeSourceJob.perform_later(@source)
```

Comment:
activejob supports globalID (http://edgeguides.rubyonrails.org/active_job_basics.html#globalid) so AR object is fine.